### PR TITLE
Update the extension builder

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: jupyterlab/index.js [hunk 1]
```diff
@@ -13,33 +15,25 @@ require('jupyterlab/lib/default-theme/index.css');
...
-  // We use the default export from es6 modules.
-  if (data.__esModule) {
-    data = data.default;
-  }
-  if (!Array.isArray(data)) {
-    data = [data];
+function getEntryPoint(entryPoint) {
+  var plugins = jupyter.require(entryPoint);
+  if (!Array.isArray(plugins)) {
+    plugins = [plugins];
```
Comment (jasongrout) [inline jupyterlab/index.js]: What happened to the esModule condition?

Comment (blink1073) [inline jupyterlab/index.js]: oh, drat.  We need the validate function to return a value then.

Comment (blink1073) [inline jupyterlab/index.js]: updated

**Comment by blink1073** [comment]
I am currently testing this PR on Windows.

**Comment by blink1073** [comment]
It worked on Windows 10 with Firefox.

**Comment by jasongrout** [comment]
This looks good to me.

**Pull request merged**